tests/model_fields/test_filefield.py

```diff
@@ -99,9 +99,10 @@ def test_move_temporary_file(self):
             "something.txt", "text/plain", 0, "UTF-8"
         ) as tmp_file:
             tmp_file_path = tmp_file.temporary_file_path()
-            Document.objects.create(myfile=tmp_file)
+            document = Document.objects.create(myfile=tmp_file)
             self.assertFalse(
-                os.path.exists(tmp_file_path), "Temporary file still exists"
+                document.myfile.storage.exists(tmp_file_path),
+                "Temporary file still exists",
```

Comment:
In this case we should leave `os.path.exists()` as this test checks that the temporary uploaded file is moved rather than copied to the destination. We should probably remove a temporary file in `InMemoryStorage._save()`, to get the same behavior with `InMemoryStorage` :thinking: